guides/v2.3/comp-mgr/install-extensions/inventory-management-installation.md

```diff
@@ -137,6 +149,10 @@ To enable Inventory Management modules:
    bin/magento cache:clean
    ```  
 
+### Previous Inventory Management versions
```

Comment:
Links not rendering (maybe GH preview error)

![screen shot 2019-02-13 at 7 38 58 am](https://user-images.githubusercontent.com/14031342/52715609-781d5700-2f62-11e9-86a1-0fef3eacd3ad.png)

Reply:
It's just a preview error in GH. That's common for page.baseurl having spaces around it. Removed spaces.